src/test/java/plugins/CompressArtifactsPluginTest.java

```diff
@@ -75,6 +77,33 @@ public void access_uncompressed_artifact_after_plugin_was_installed() {
         assertThat(build, not(hasCompressedArtifacts()));
     }
 
+    @Test @Bug("JENKINS-27042")
+    @WithPlugins("compress-artifacts")
+    public void archiveLargerThan4GInTotal() throws Exception {
+        configureArtifactCompression();
+
+        FreeStyleJob job = jenkins.jobs.create();
+        job.configure();
+        job.addPublisher(ArtifactArchiver.class).includes("*");
+        job.addBuildStep(ShellBuildStep.class).command( // Generate archive larger than 4G
+                "wget $JENKINS_URL/jnlpJars/jenkins-cli.jar -O stuff.jar; for i in {0..7000}; do cp -l stuff.jar stuff.${i}.jar; done"
```

Comment:
Instead of relying on wget being installed you could do something like the other archive tests: https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/test/java/core/ArtifactsTest.java#L45